src/validation.h

```diff
@@ -120,9 +120,9 @@ static const unsigned int AVG_FEEFILTER_BROADCAST_INTERVAL = 10 * 60;
 /** Maximum feefilter broadcast delay after significant change. */
 static const unsigned int MAX_FEEFILTER_CHANGE_DELAY = 5 * 60;
 /** Block download timeout base, expressed in millionths of the block interval (i.e. 10 min) */
-static const int64_t BLOCK_DOWNLOAD_TIMEOUT_BASE = 1000000;
+static const int64_t BLOCK_DOWNLOAD_TIMEOUT_BASE = 10000000;
```

Comment:
Would recommend 2 minutes, and at absolute most 5 minutes. 10 minutes is a lot of time for a 1MB block and exhausts connections that can likely be better served by a reconnect to another node?